### DISCUSSION TITLE
How to add annotations to highstock?

### DISCUSSION THREAD

**Discussion by wildanhkm** [richardeschloss/nuxt-highcharts#57] [Q&A]
I have created a highstock chart using this nuxt-highcharts package, I do have a question on how to add annotations to highstock chart?

**Comment by richardeschloss** [comment]
Highcharts annotations requires the annotations module. To load that module with nuxt-highcharts, use the `modules` prop:

```js
<highstock :options="chartOpts" :modules="['annotations']" />
```

Where `chartOpts` would specify the annotations (and the series data would choose what annotations to use). See [Basic Annotations Example](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/annotations/basic/) for an idea of how to use it.
